ament_cmake_core/cmake/core/package_xml_2_cmake.py

```diff
@@ -16,8 +16,10 @@
 
 import argparse
 from collections import OrderedDict
+from os import environ
```

Comment:
Nitpick: usually `import os` is used (similar to `sys` next line) and then used as `os.environ`.